dwave/preprocessing/include/dwave/presolve.hpp

```diff
@@ -138,6 +141,11 @@ class Presolver {
     /// This clears the model from the presolver.
     model_type detach_model();
 
+    void load_taskflow_one_time();
```

Comment:
Not sure if this is intermediate or final design, I think ultimately we want `apply()` to just run the taskflow, whatever is there. And then the `load_` methods will determine what is in that flow.